### PR TITLE
Update to support OpenShift GitOps 1.11

### PR DESCRIPTION
**Pull request by sabre1041**
Update to support OpenShift GitOps 1.11

### CODE REVIEW

**Comment by gnunn1** [comment]
LGTM however I'll leave to the others to comment on changing `patch-channel` to `patch-subscription` since everywhere else AFAIK we are using `patch-channel`. Or maybe an oppty to start moving this into inline patches in kustomization.yaml?

**Comment by sabre1041** [comment]
> LGTM however I'll leave to the others to comment on changing `patch-channel` to `patch-subscription` since everywhere else AFAIK we are using `patch-channel`. Or maybe an oppty to start moving this into inline patches in kustomization.yaml?

We need to patch more than the channel starting in 1.11 since it deploys to a unique namespace

**Comment by pittar** [comment]
@gnunn1 , I'd be happy to start moving these patches inline.  Maybe I can find some time over the holidays :)

**Comment by pittar** [comment]
@sabre1041 , if we moved the patch inline, would that solve the naming issue?

**Comment by sabre1041** [comment]
> @sabre1041 , if we moved the patch inline, would that solve the naming issue?

I do not understand the the issue with changing the name of the patch. It is performing more than just patching the channel

**Comment by gnunn1** [comment]
I'm ok with the name change given @sabre1041 comment.

With the change in namespace do we need a readme comment about updating from other overlay versions as the namespace change will break things if you don't have pruning enabled in Argo CD?

File: openshift-gitops-operator/README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The current *overlays* available are for the following channels:
 * [gitops-1.8](operator/overlays/gitops-1.8)
 * [gitops-1.9](operator/overlays/gitops-1.9)
 * [gitops-1.10](operator/overlays/gitops-1.10)
+* [gitops-1.11](operator/overlays/gitops-1.10)
```
Comment (adetalhouet) [inline openshift-gitops-operator/README.md:17]: typo. Should be `[gitops-1.11](operator/overlays/gitops-1.11)` you currently have `-1.10`

**Review by adetalhouet — COMMENTED** [review]
There is a typo in the readme; beside that, look good to me.
Moving the operator to its own namespace is a good idea ([I did this a year ago](https://github.com/adetalhouet/ocp-gitops/commit/0525f63cbada963150e7f625c3c4956931cef6c0) and it removed a lot of OLM related issues.

**Comment by adetalhouet** [comment]
I'll let @gnunn1 or @pittar merge this given the outstanding discussion. From my standpoint, it is good to go.

**Comment by pittar** [comment]
I'd never argue against @sabre1041  ;)

**Pull request merged**